### ISSUE TITLE
Missing translation files

### ISSUE THREAD

**Issue by josch1710** [bitwiseworks/qtbase-os2#131]
**Describe the bug**
There are no QT translation files in the RPM packages

**To Reproduce**
/@unixroot/usr/share/qt5/translations is empty

**Expected behavior**
Lots of *.qm files, eg. qtbase_de.qm

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: ArcaOS 5
 - QT5 5.13.1-1

**Comment by SilvanScherrer** [comment]
is this still the case with latest Qt5?

**Comment by dmik** [comment]
Sure it's still the case. Qt translations are gathered into a separate repository (https://code.qt.io/cgit/qt/qttranslations.git/tree/?h=v5.15.2 is for our latest release) and we never built it. Should be not difficult to cook an RPM for that repo once someone sponsors. Will leave it open till then (the respective Fedora .spec repo is this one https://src.fedoraproject.org/rpms/qt5-qttranslations).

In the mean time, one can compile `.ts` files into `.qm` files manually with the `lrelease` command line tool (or with Qt Linguist — we provide both) and then put them to the respective directory (`/@unixroot/usr/share/qt5/translations` I believe). Also https://wiki.qt.io/Qt_Localization contains some useful links.

**Comment by josch1710** [comment]
Yes, one can compile them. So, I bundle them with my FeatherPad and FeatherNotes builds, so that they can work correctly, when shortcuts get translated, e.g. in German CTRL => STRG.
